ccd-definition/AuthorisationCaseField/la-solicitor.json

```diff
@@ -778,7 +785,7 @@
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "CaseFieldID": "amountToPay",
     "UserRole": "[LASOLICITOR]",
-    "CRUD": "CRUD"
+    "CRUD": "CRU"
```

Comment:
Why remove the `D` 😉 😕 
Aren't you going to want to remove the field?

Reply:
Checked case submission and looks like it's not necessary when we populate field in about-to-start event and remove it in about-to-submit